f12-opshin-audit-closeout-report/opshin-close-out-report.typ

```diff
@@ -0,0 +1,254 @@
+    #line(length: 100%) // Add a line above the footer
+    #align(center)[*Anastasia Labs* \ Project Close-out Report]
+  ],
+  footer-descent: 20%,
+)
+
+// This command can be turned on to justify the text #set par(justify: true)
+
+#show link: underline
+#show outline.entry.where(level: 1): it => {
+  v(12pt, weak: true)
+  strong(it)
+}
+
+// Initialize page counter
+#counter(page).update(0)
+
+#set page(footer: [
+  #set text(11pt)
+  #line(length: 100%) // Add a line above the footer
+  #align(center)[*Anastasia Labs* \ Project Close-out Report]
+  #place(right, dy: -7pt)[#counter(page).display("1/1", both: true)]
+])
+#v(100pt)
+
+// Configure the outline depth and indentation
+#outline(depth: 2, indent: 1em)
+
+// Page break
+#pagebreak()
+#set terms(separator: [: ], hanging-indent: 40pt)
+#v(20pt)
+/ Project Name: Opshin Audit
+/ URL: #link(
+    "link",
+  )[https://projectcatalyst.io/funds/12/cardano-open-developers/opshin-audit]
+
+#v(10pt)
+= List of KPIs
+#v(10pt)
+== Challenge KPIs
+#v(10pt)
+- *Improve Smart Contract Security:* By conducting an in-depth audit of the OpShin
+  language, the project directly addresses security risks associated with smart
+  contract development. The expected outcome is a measurable reduction in critical
+  and high-severity vulnerabilities, making DApps built with OpShin safer for
+  users.
+
+- *Enhance Developer Knowledge and Best Practices:* The project provides clear,
+  actionable best practices and development guidelines for using OpShin
+  effectively. This helps elevate the quality of code across the ecosystem and
+  supports developer onboarding.
+
+- *Drive Ecosystem Trust and Maturity:* The public audit and transparent
+  communication of findings will reinforce OpShin’s position as a secure and
+  trustworthy platform for smart contract development. By identifying edge cases
+  and potential problems early in OpShin, the audit enables developers to avoid
+  time-consuming debugging and security rework.
+#pagebreak()
+#v(20pt)
+== Project KPIs
+#v(40pt)
+- *Reduction in Critical Vulnerabilities:* As part of our comprehensive audit of
+  the OpShin language, we identified 9 critical and 16 major vulnerabilities
+  within the codebase. These findings highlight previously unaddressed security
+  risks that could impact smart contract reliability and user safety. By surfacing
+  these issues early and providing detailed recommendations, the audit strengthens
+  the foundation for secure smart contract development using OpShin. This outcome
+  not only enhances developer confidence in using OpShin but also helps reduce the
+  risk of exploit-prone contracts being deployed on mainnet.
+
+- *Audit Delivered with Categorized Findings:* The audit report provides a
+  detailed breakdown of each identified issue, categorized by Security,
+  Performance, Maintainability, and Usability, and classified by severity as
+  Critical, High, Medium, or Informational. Each finding includes an assessment of
```

Comment:
Inconsistent severities: here we have "critical", "high", but in the previous point (Reduction in Critical Vulnerabilities) we have "critical" and "major".